### PR TITLE
Port over all as-is changes

### PR DESCRIPTION
**Pull request by timnlupo**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     author="Amazon",
     description="Amazon Software License",
     packages=["jupyterlab_pullrequests"],
-    install_requires=["notebook"],
+    install_requires=["notebook", "nbdime", "asynctest"],
```
Comment (jaipreet-s) [inline setup.py]: Async test should not be a library dependency. Add it as a "Test" dependency as per https://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=tests_require#new-and-changed-setup-keywords

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    - /home/travis/.yarn-cache/
+install: pip install pytest "jupyterlab<1.0.0"
+script:
+  - yarn
+  - yarn build
+  - pip install .
+  - jupyter serverextension enable --py jupyterlab_pullrequests
+  - jupyter labextension install .
+  - pytest tests
+  - yarn run lint
```
Comment (jaipreet-s) [inline .travis.yml]: Consider logically grouping these commands

- All yarn/frontend validations
- All python/backend tests/validations
- Sanity test for installing the extension

File: src/components/diff/PlainDiffComponent.tsx [hunk 3]
```diff
@@ -6,6 +6,36 @@ import * as React from "react";
 import ReactResizeDetector from 'react-resize-detector';
 import { PullRequestCommentThreadModel, PullRequestFileModel, PullRequestPlainDiffCommentThreadModel } from "../../models";
 
+import * as monacoCSS from 'file-loader!../../../lib/JUPYTERLAB_FILE_LOADER_jupyterlab-pullrequests-css.worker.bundle.js';
```
Comment (jaipreet-s) [inline src/components/diff/PlainDiffComponent.tsx]: It's hard to tell which part of this came from `jupyterlab-monaco` and which part we've added. Could you add clear comments around the parts that have been ported over?

**Comment by blink1073** [comment]
For reference: https://github.com/jupyterlab/team-compass/issues/8

**Comment by timnlupo** [comment]
@blink1073 Can you enable Travis on the `jupyterlab/pull-requests` repo? I added a `.travis.yml` and verified all was working on my repo.

**Comment by blink1073** [comment]
Done!

**Pull request merged**